client/webserver/site/src/js/markets.ts

```diff
@@ -1097,8 +1097,8 @@ export default class MarketsPage extends BasePage {
     page.lotSize.textContent = Doc.formatCoinValue(mkt.cfg.lotsize, mkt.baseUnitInfo)
     page.rateStep.textContent = Doc.formatCoinValue(mkt.cfg.ratestep / rateConversionFactor)
 
-    if (!baseAsset?.wallet || !quoteAsset?.wallet) Doc.setVis(true, page.noWallet)
-    else this.balanceWgt.setWallets(host, base, quote)
+    this.displayMessageIfMissingWallet()
+    this.balanceWgt.setWallets(host, base, quote)
```

Comment:
This change needs justification. Before, we were only calling `setWallets` if we knew we had both wallets. Why is this being changed?

Reply:
` this.balanceWgt.setWallets(host, base, quote)` handles a missing `asset.wallet` properly by displaying the "Add a wallet" btn in place of the balance widget. If one or both wallet is missing, the expected "Add a Wallet" btn will be displayed for that wallet.

Reply:
Maybe it's OK. I don't know. The question is why is it being changed?

Reply:
> The question is why is it being changed?

I initially added this line of code in https://github.com/decred/dcrdex/pull/2458 to fix display some display issues.
> - Ensure there are visible changes when user clicks on another market instead
of waiting for order book note which might delay. These visible
changes include: ....., **setting the balance widget wallets**.

But now I'm looking at it again, this.balanceWgt.setWallets(host, base, quote) handles a missing asset.wallet properly so I think it's the correct thing to do.

Reply:
You're missing the point. Don't fix things that aren't broken. You're introducing behavior changes that have nothing to do with this PR.